content/resources/data.toml

```
@@ -10,12 +10,30 @@ description = "A curated list of Rust code and resources, inspired by other awes
 categories = ["lists"]
 homepage_url = "//github.com/kud1ing/awesome-rust"
 
+[[items]]
+name = "Awesome wgpu"
+description = "A curated list of wgpu code and resources"
+categories = ["lists"]
+homepage_url = "github.com/rofrol/awesome-wgpu"
```

This either needs a `//` or a `https://` at the front - it's currently getting resolved as a relative link (another thing to add to the CI check, maybe?)

```suggestion
homepage_url = "//github.com/rofrol/awesome-wgpu"
```